Update azure-stack-tutorial-app-service.md

apwestgarth opened the pull request:
Remove mention of mobile apps as they are not included in GA release of Azure App Service on Azure Stack

azurecla commented:
Hi __@rboucher__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

apwestgarth committed:
Update azure-stack-tutorial-app-service.md

Remove mention of mobile apps as they are not included in GA release of Azure App Service on Azure Stack
